packages/cascara/src/structures/AdminStructure/components/Main/Main.js

```diff
@@ -46,7 +46,7 @@ const Main = ({
   const { className, ...props } = rest;
   const tabs = useTabState();
   return (
-    <Role as={as} className={cx(className, 'Main')}>
+    <Role as={as} className={cx('Main', className)}>
```

Comment:
```suggestion
    <Role {...rest} as={as} className={cx('Main', rest.className)}>
```
We are missing the rest spread here.